jbmc/src/java_bytecode/java_bytecode_language.h

```diff
@@ -62,6 +63,10 @@ Author: Daniel Kroening, kroening@kroening.com
   "                              (byte, short, int, long, float, double) to an entry\n" /* NOLINT(*) */ \
   "                              point within the given range; lower bound L\n" /* NOLINT(*) */ \
   "                              and upper bound U must be integers\n" /* NOLINT(*) */ \
+  " --java-assume-inputs-integral\n" \
+  "                              only initialize numerical primitive-typed parameter\n" /* NOLINT(*) */ \
+  "                              to an entry point that is integral, not fractional;\n" /* NOLINT(*) */ \
+  "                              results in float and double having int value\n" /* NOLINT(*) */ \
```

Comment:
:no_entry_sign: Also needs documentation here: https://github.com/diffblue/test-gen/blob/develop/doc/cl-options.md
with a short example, ideally.
The limitation (arrays) should also be stated there.

Reply:
Definitely, that will be part of the test-gen PR.

Reply:
I added the limitation on arrays

Reply:
Hmmm... of course, that PR is indeed on cbmc... :man_facepalming: